StyleSheet.css

```diff
@@ -21,6 +21,62 @@ nav {
   padding: 6px 24px;
 }
 
+.nav-wrapper {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  height: 100vh;
+  width: 100vw;
+  -moz-transition: all 0.3s ease-in;
+  -ms-transition: all 0.3s ease-in;
+  -o-transition: all 0.3s ease-in;
+  -webkit-transition: all 0.3s ease-in;
+  transition: all 0.3s ease-in;
+}
+
+.nav-wrapper.open {
+  left: 0;
+}
+
+.nav-bg {
+  background-color: #6070ffdc;
+  -webkit-backdrop-filter: blur(20px);
+  backdrop-filter: blur(20px);
```

Comment:
- [ ] To match the Figma design, kindly make sure one party of the mobile menu design can not be visible before clicking the `hamburger` menu. Try to update your design and stick as much as possible to the expected design.
![Screenshot from 2022-12-19 22-34-52](https://user-images.githubusercontent.com/83042742/208528850-dabc0b0f-dfcf-4f40-aa8c-729676ba812d.png)